From: Gordon Ball <gordon@chronitis.net>
Date: Tue, 12 Nov 2019 10:26:08 +0000
Subject: Patch out obsolete grffile from latex template

This removes references to grffile, which is now a no-op stub. See
https://github.com/jupyter/nbconvert/issues/1140

Patch from Christian Christensen.
---
 share/jupyter/nbconvert/templates/latex/base.tex.j2 | 15 ---------------
 1 file changed, 15 deletions(-)

--- nbconvert.orig/share/jupyter/nbconvert/templates/latex/base.tex.j2
+++ nbconvert/share/jupyter/nbconvert/templates/latex/base.tex.j2
@@ -57,21 +57,6 @@
     \fi
 
     \usepackage{fancyvrb} % verbatim replacement that allows latex
-    \usepackage{grffile} % extends the file name processing of package graphics
-                         % to support a larger range
-    \makeatletter % fix for old versions of grffile with XeLaTeX
-    \@ifpackagelater{grffile}{2019/11/01}
-    {
-      % Do nothing on new versions
-    }
-    {
-      \def\Gread@@xetex#1{%
-        \IfFileExists{"\Gin@base".bb}%
-        {\Gread@eps{\Gin@base.bb}}%
-        {\Gread@@xetex@aux#1}%
-      }
-    }
-    \makeatother
     \usepackage[Export]{adjustbox} % Used to constrain images to a maximum size
     \adjustboxset{max size={0.9\linewidth}{0.9\paperheight}}
 
